statsmodels/tsa/tests/test_arima.py

```diff
@@ -497,95 +504,153 @@ def setup_class(cls):
+                     1342, 1589, 2361,
+                     3452, 2659, 2857, 3255, 3322, 2852, 2964, 3132, 3033,
+                     2931, 2636, 2818,
+                     3310, 3396, 3179, 3232, 3543, 3759, 3503, 3758, 3658,
+                     3425, 3053, 2620,
+                     1837, 923, 712, 1054, 1376, 1556, 1498, 1523, 1088, 728,
+                     890, 1413, 2524,
+                     3295, 4097, 3993, 4116, 3874, 4074, 4142, 3975, 3908,
+                     3907, 3918, 3755,
+                     3648, 3778, 4293, 4385, 4360, 4352, 4528, 4365, 3846,
+                     4098, 3860, 3230,
+                     2820, 2916, 3201, 3721, 3397, 3055, 2141, 1623, 1825,
+                     1716, 2232, 2939,
+                     3735, 4838, 4560, 4307, 4975, 5173, 4859, 5268, 4992,
+                     5100, 5070, 5270,
+                     4760, 5135, 5059, 4682, 4492, 4933, 4737, 4611, 4634,
+                     4789, 4811, 4379,
+                     4689, 4284, 4191, 3313, 2770, 2543, 3105, 2967, 2420,
+                     1996, 2247, 2564,
+                     2726, 3021, 3427, 3509, 3759, 3324, 2988, 2849, 2340,
+                     2443, 2364, 1252,
+                     623, 742, 867, 684, 488, 348, 241, 187, 279, 355, 423,
+                     678, 1375, 1497,
+                     1434, 2116, 2411, 1929, 1628, 1635, 1609, 1757, 2090,
+                     2085, 1790, 1846,
+                     2038, 2360, 2342, 2401, 2920, 3030, 3132, 4385, 5483,
+                     5865, 5595, 5485,
+                     5727, 5553, 5560, 5233, 5478, 5159, 5155, 5312, 5079,
+                     4510, 4628, 4535,
+                     3656, 3698, 3443, 3146, 2562, 2304, 2181, 2293, 1950,
+                     1930, 2197, 2796,
+                     3441, 3649, 3815, 2850, 4005, 5305, 5550, 5641, 4717,
+                     5131, 2831, 3518,
+                     3354, 3115, 3515, 3552, 3244, 3658, 4407, 4935, 4299,
+                     3166, 3335, 2728,
+                     2488, 2573, 2002, 1717, 1645, 1977, 2049, 2125, 2376,
+                     2551, 2578, 2629,
+                     2750, 3150, 3699, 4062, 3959, 3264, 2671, 2205, 2128,
+                     2133, 2095, 1964,
+                     2006, 2074, 2201, 2506, 2449, 2465, 2064, 1446, 1382, 983,
+                     898, 489, 319,
+                     383, 332, 276, 224, 144, 101, 232, 429, 597, 750, 908,
+                     960, 1076, 951,
+                     1062, 1183, 1404, 1391, 1419, 1497, 1267, 963, 682, 777,
+                     906, 1149, 1439,
+                     1600, 1876, 1885, 1962, 2280, 2711, 2591, 2411])
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
-        res = ARMA(data, order=(4,1)).fit(start_ar_lags=5, disp=-1)
+        ARMA(data, order=(4, 1)).fit(start_ar_lags=5, disp=-1)
 
 
 class Test_ARIMA101(CheckArmaResultsMixin):
     # just make sure this works
```

Comment:
does this indicate this is a smoke test?

Reply:
outdated.